src/Momento.Sdk/ISimpleCacheClient.cs

```diff
@@ -195,6 +195,37 @@ public interface ISimpleCacheClient : IDisposable
     /// <inheritdoc cref="SetAsync(string, byte[], byte[], TimeSpan?)"/>
     public Task<CacheSetResponse> SetAsync(string cacheName, string key, byte[] value, TimeSpan? ttl = null);
 
+    /// <summary>
+    /// <para>Sets the value of of the given key and field if the key does not exists in cache</para>
+    ///
+    /// <para>Associates the given key with the given value. If a value for the key is
+    /// already present it is not replaced with the new value.</para>
+    /// </summary>
+    /// <param name="cacheName">Name of the cache to store the key and field in.</param>
+    /// <param name="key">The key to set.</param>
+    /// <param name="field">The value to be stored.</param>
+    /// <param name="ttl">TTL for the field in cache. This TTL takes precedence over the TTL used when initializing a cache client. Defaults to client TTL.</param>
+    /// <returns>Task representing the result of the cache operation.</returns>
+    public Task<CacheSetIfNotExistsResponse> SetIfNotExistsAsync(string cacheName, string key, string field, TimeSpan? ttl = null);
```

Comment:
I think `field` should be named `value` here, and you probably want a second signature of this that will accept `byte[]`?

Reply:
Yup, will fix, thanks!

Reply:
Fixed